code/modules/antagonists/role_preference/role_antagonists.dm

```diff
@@ -0,0 +1,32 @@
+/datum/role_preference/antagonist/blood_brother
+	name = "Blood Brother"
+
+/datum/role_preference/antagonist/blood_cultist
+	name = "Blood Cultist"
+
+/datum/role_preference/antagonist/clock_cultist
+	name = "Clock Cultist"
+
+/datum/role_preference/antagonist/devil
+	name = "Devil"
+
+/datum/role_preference/antagonist/revolutionary
+	name = "Head Revolutionary"
+
+/datum/role_preference/antagonist/heretic
+	name = "Heretic"
+
+/datum/role_preference/antagonist/hivemind_host
+	name = "Hivemind Host"
+
+/datum/role_preference/antagonist/incursionist
+	name = "Incursionist"
+
+/datum/role_preference/antagonist/excommunicate
+	name = "Excommunicated Syndicate Agent"
+
+/datum/role_preference/antagonist/gangster
+	name = "Gangster"
+
+/datum/role_preference/antagonist/internal_affairs
+	name = "Internal Affairs Agent"
```

Comment:
It's way less maintainable that way, in TGUI Prefs these have a ton of extra code attached to them. Right now they're basically placeholders. It's best to keep them organized in one or two files since the order they're declared also determines display order and it's easier to locate them.

Reply:
Acceptable, resolving